Conf/com.northpolesec.santa.bundleservice.plist

```diff
@@ -22,6 +22,8 @@
 	<string>Interactive</string>
```

Comment:
s/Interactive/Adaptive

Reply:
I am going to save that change for a future PR so I can test it a bit more thoroughly.